element_array_ephys/probe.py

```diff
@@ -100,114 +100,59 @@ def create_neuropixels_probe(probe_type: str = "neuropixels 1.0 - 3A"):
                 white_spacing=0,
-                col_count=2,
+                col_count_per_shank=2,
                 shank_count=1,
                 shank_spacing=250,
             ),
             "neuropixels 2.0 - MS": dict(
-                site_count=1280,
+                site_count_per_shank=1280,
                 col_spacing=32,
                 row_spacing=15,
                 white_spacing=0,
-                col_count=2,
+                col_count_per_shank=2,
                 shank_count=4,
                 shank_spacing=250,
             ),
         }
 
-        def build_electrodes(
-            site_count: int,
-            col_spacing: float,
-            row_spacing: float,
-            white_spacing: float,
-            col_count: int,
-            shank_count: int,
-            shank_spacing: float,
-        ) -> dict:
-            """Builds electrode layouts.
-
-            Args:
-                site_count (int): site count per shank
-                col_spacing (float): (um) horizontal spacing between sites
-                row_spacing (float): (um) vertical spacing between columns
-                white_spacing (float): (um) offset spacing
-                col_count (int): number of column per shank
-                shank_count (int): number of shank
-                shank_spacing (float): spacing between shanks
-            """
-            row_count = int(site_count / col_count)
-            x_coords = np.tile(
-                np.arange(0, col_spacing * col_count, col_spacing), row_count
-            )
-            y_coords = np.repeat(np.arange(row_count) * row_spacing, col_count)
-
-            if white_spacing:
-                x_white_spaces = np.tile(
-                    [white_spacing, white_spacing, 0, 0], int(row_count / 2)
-                )
-                x_coords = x_coords + x_white_spaces
-
-            shank_cols = np.tile(range(col_count), row_count)
-            shank_rows = np.repeat(range(row_count), col_count)
-
-            npx_electrodes = []
-            for shank_no in range(shank_count):
-                npx_electrodes.extend(
-                    [
-                        {
-                            "electrode": (site_count * shank_no) + e_id,
-                            "shank": shank_no,
-                            "shank_col": c_id,
-                            "shank_row": r_id,
-                            "x_coord": x + (shank_no * shank_spacing),
-                            "y_coord": y,
-                        }
-                        for e_id, (c_id, r_id, x, y) in enumerate(
-                            zip(shank_cols, shank_rows, x_coords, y_coords)
-                        )
-                    ]
-                )
-
-            return npx_electrodes
-
-        electrodes = build_electrodes(**neuropixels_probes_config[probe_type])
         probe_type = {"probe_type": probe_type}
+        electrode_layouts: list[dict] = build_electrode_layouts(
```

Comment:
What's the benefit of adding `list[dict]` typing here? For functions, it might help someone understand the code later, but I don't think we're in the habit of doing so when grabbing results